### DISCUSSION TITLE
HmIP-ESI-GAS fehlende Werte in HomeAssistant - Kanal 2

### DISCUSSION THREAD

**Discussion by deGrasi** [SukramJ/aiohomematic#1511] [Q&A]
Bei meinem Gaszähler ist mir aufgefallen, dass ein Kanal nicht ausgelesen wird, der zum Beispiel den Zählerstand übermittelt:

In RaspberryMatic sieht es so aus:
![c98ef577a556933c75a602cc24c6061b53264971](https://github.com/danielperna84/hahomematic/assets/107140531/bf1de4a3-1bf6-406d-b690-93feb470c143)

In Home Assistant sieht es so aus:
![0dd2755032e5b5ea76b03f6352898b27955c89c0](https://github.com/danielperna84/hahomematic/assets/107140531/f5ac7aba-a89b-4cb2-9a25-e322f4d27f91)

Würde mir wünschen, wenn der Zählerstand ebenfalls auslesbar in HA zur Verfügung steht.

Ich habe keine Ahnung, ob ich hier an der richtigen Stelle dafür bin. Über Hilfe bin ich euch sehr dankbar.

**Comment by SukramJ** [comment]
Bitte die Suche verwenden: #1479
